bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,28 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
```

Comment:
@hvelarde why are you removing the buildout cache here? That's actually quite useful.